### DISCUSSION TITLE
Unable to create new project

### DISCUSSION THREAD

**Discussion by mortezabonak** [adonisjs/core#2049] [General]
hi
I want to create a project but got some errors
![Screenshot (176)](https://user-images.githubusercontent.com/33400847/102178013-e82bf980-3eb9-11eb-912f-7788062ca2b6.png)

**Comment by HappyLDE** [comment]
Try yarn or if you used yarn try npm?

**Reply by mortezabonak** [reply]
I did both, but neither worked

**Reply by thetutlage** [reply]
This is an issue occurred coz of an old version of `create-adonis-ts-app` lying in your global modules. I believe the following command should ignore the cache

```sh
npx --ignore-existing create-adonis-ts-app <project-name>
```

**Reply by mortezabonak** [reply]
fixed
removed the old package and then reinstalled it
